eureka2-ext/eureka2-eureka1-rest-api/src/main/java/com/netflix/eureka2/eureka1/rest/query/Eureka2RegistryViewCache.java

```diff
@@ -92,6 +87,20 @@ public Applications findApplicationsBySecureVip(final String secureVipAddress) {
         return findApplicationsBy(secureVipAddress, Interests.forSecureVips(secureVipAddress), secureVipMap);
     }
 
+    private RegistryFetch getLatestFullFetch() {
+        if (fullFetchWithDeltaView.get() == null) {
```

Comment:
Why not inititalize this hot or just use synchronized? This seems more complex than needed.